### DISCUSSION TITLE
Is there a way to show just one day summary?

### DISCUSSION THREAD

**Discussion by GinoReiji** [mcndt/obsidian-toggl-integration#123] [Q&A]
I would like to make a template for my daily note to show the summary just for that day. The TODAY would only work for the current day, yes? If I look at my daily note on the 2023-03-25, how do I make so that that note appeared the toggl report on that day (2023-03-25)?

**Comment by adamtajti** [comment]
I used this snippet in my daily note template @GinoReiji. It works alright.

```md

## Time Tracking

```toggl
LIST FROM {{date}} TO {{date}}
GROUP BY PROJECT
SORT DESC```

```

**Reply by ricraftis** [reply]
I can't get this to work.

```toggl
LIST FROM {{date}} TO {{date}} 
GROUP BY PROJECT 
SORT DESC
```

Error message:

"{{date}}" is not a keyword, ISO-formatted date, decimal number or string. Hint: strings must be wrapped in quotation marks, dates must be formatted as YYYY-MM-DD and numbers should have no leading zeros.

**Reply by mayukhc** [reply]
Add it in the daily note template, the date will get filled in when the note's getting created.

**Reply by ricraftis** [reply]
Yes, got that sorted. Should have posted a reply to confirm. Any ideas on how to get the last 7 days one to remain fixed based on date of note?
